src/components/common-components/Notifications.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+    <ul id="messages" v-for="message in messages" :key="message.id">
+        <li>{{message.message}}</li>
+    </ul>
+</template>
+
+<script lang="ts">
+import { defineComponent, getCurrentInstance, inject } from 'vue'
+import {ActionTypes} from '@/store/modules/auth/actions'
+import {mapActions, mapGetters} from 'vuex'
+
+
+export default defineComponent({
+    computed:{
+        ...mapGetters({
+            user: 'getUser',
+            user_company: 'getUserCompany',
+            messages: 'getNotifications'
+        })
+    },
+    methods:{
+        ...mapActions({
+            userdata: ActionTypes.USER_DATA
+        })
+    },
+    beforeMount: async function(){
+        await this.userdata();
+        this.$socket.emit('client_info', {id: this.user.id, company: this.user_company.id});
```

Comment:
Remove socket code from component. It will be in the plugin.

Reply:
user data is not available in plugin.